index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width , initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>KMarketer: We Take Care of the Future</title>
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+
+<body>
+    <form action="/action_page.php" method="get">
```

Comment:
Why the from wraps the whole block? 
You don't need the form here at all as you don't have any form elements here